src/dapps/ens-manager-dapp/components/manage/ManageMulticoins.vue

```diff
@@ -39,45 +39,70 @@ export default {
   },
   data() {
     const errors = {};
+    const coins = {};
     for (const type in this.multicoin) {
       errors[type] = '';
+      coins[type] = this.multicoin[type].value;
     }
     return {
-      setCoins: [],
+      setCoins: coins,
       errors: errors
     };
   },
   computed: {
     coins() {
-      const coins = [];
-      for (const type in this.multicoin) {
-        coins.push(this.multicoin[type]);
-      }
-      return coins;
+      return Object.keys(this.setCoins);
     },
     hasError() {
       return (
         Object.values(this.errors).filter(item => {
           if (item !== '') return item;
         }).length > 0
       );
+    },
+    valid() {
+      const hasValues = Object.values(this.setCoins).filter(item => {
+        if (item && item.value !== '') return item;
+      });
+
+      return hasValues.length > 0 && !this.hasError;
+    }
+  },
+  watch: {
+    multicoin: {
+      handler: function (newVal) {
+        if (newVal) {
+          for (const type in newVal) {
+            this.setCoins[type] = newVal[type].value;
+            this.errors[type] = '';
+          }
+        }
+      },
+      deep: true,
+      immediate: true
     }
   },
   methods: {
-    setCoin(value, id) {
+    validateCoin(value, id) {
       const coin = this.coins[id];
-      if (coin.validator.validate(value)) {
-        const newObj = Object.assign({}, coin, { value: value });
-        this.setCoins.push(newObj);
-      } else {
+      if (value && !this.multicoin[coin].validator.validate(value)) {
         if (value !== '') {
-          this.errors[coin.symbol] = this.$t('ens.ens-resolver.invalid-addr', {
-            coin: coin.name
+          this.errors[coin] = this.$t('ens.ens-resolver.invalid-addr', {
+            coin: coin
           });
         } else {
-          this.errors[coin.symbol] = '';
+          this.errors[coin] = '';
         }
       }
+    },
+    submit() {
+      /** Creates a no reference clone to be submitted to the contract */
+      const copyMulticoin = Object.keys(this.multicoin).map(item => {
+        const newItem = Object.assign({}, _.clone(this.multicoin[item]));
+        newItem.value = this.setCoins[item];
```

Comment:
only add it if the old value changed, if not itll be an expensive tx since itll set whole bunch of 000...000 addresses